src/actions/adminAction.js

```diff
@@ -27,31 +26,27 @@ export const createAdmin = (adminInfo) => async (dispatch) => {
 export const loginAdmin = (adminInfo, history) => async (dispatch) => {
   try {
     const res = await axios.post(`${BASE_URL}/auth/login/`, adminInfo)
-   dispatch(setRequestStatus(false));
-   if (res.status === 200) {
+    dispatch(setRequestStatus(false));
+    if (res.status === 200) {
+      dispatch(setRequestStatus(true));
 
-     const token = res.data.token;
-     dispatch(setRequestStatus(true));
+      // update state with user
+      localStorage.setItem('userId', res.data.user)
+      dispatch(setCurrentUser(res.data.user._id));
 
-     localStorage.setItem("jwtToken", (token));
-     setAuthToken(token);
+      // update localStorage with admin status 
+      localStorage.setItem('username', `${res.data.user.name.firstName} ${res.data.user.name.lastName}`)
```

Comment:
Local storage is not that secure!!
Try to come up with some secure approach!!
As of now, you can go with it but it's not acceptable at the production level

Reply:
Setting names in local storage won't do any harm as the protected resources sent from the server need that token in cookie.